addons/payment/data/payment_method_data.xml

```diff
@@ -3433,7 +3455,7 @@
 
     <record id="payment_method_wallets_india" model="payment.method">
         <field name="name">Wallets India</field>
-        <field name="code">wallets_india</field>
+        <field name="code">wallet</field>
```

Comment:
I don't think we should do that. This payment method is specific to India, and there are other eWallet PMs out there that are specific to other countries. If there is a Brazilian eWallet PM, we won't be able to name it `wallet`; it'll have to be `wallets_brazil` with a mapping anyway. We should apply the same logic to `wallets_india`.

Reply:
Got it! :100: